pandas/io/gbq.py

```diff
@@ -516,7 +613,7 @@ def to_gbq(dataframe, destination_table, project_id, chunksize=10000,
     else:
         table.create(table_id, table_schema)
 
-    connector.load_data(dataframe, dataset_id, table_id, chunksize, verbose)
+    connector.load_data(dataframe, dataset_id, table_id, chunksize)
 
 
 def generate_bq_schema(df, default_type='STRING'):
```

Comment:
Hey! Guys! I didn't touch those two functions `generate_bq_schema` nor `_generate_bq_schema`. Can we finally close this change? Plss